ITSM Connector Release Notes.md

```diff
@@ -0,0 +1,24 @@
+# VIAcode ITSM connector for Azure Release Notes
+
+<!-- TOC -->
+- [VIAcode ITSM connector for Azure initial release](#viacode-itsm-connector-for-azure-initial-release)  
+<!-- TOC END -->
+
+## VIAcode ITSM connector for Azure initial release
+
+- Creation of tickets with articles in VIAcode Incident Management System for:
+    - Azure Monitor alerts
+    - Azure Security Center alerts
+    - Azure Security Center recommendations
+    - Azure Advisor recommendations
+    - Budget alerts
```

Comment:
"Cost Management Alerts"  is more correct name ?

Reply:
As far as I understand they are different things.
As written in documentation "There are three types of cost alerts: budget alerts, credit alerts, and department spending quota alerts."
So budget alerts is a special case of cost alerts.
As far as I know we currently support only budget alerts.
So I thought that it would be more precise to tell that we support budget alerts...

Reply:
Got you, may be then: "Cost Alert (budget based)"